lib/Dialect/FIRRTL/Transforms/IMConstProp.cpp

```diff
@@ -417,6 +416,9 @@ void IMConstPropPass::markRegResetOp(RegResetOp regReset) {
   if (enable.isOverdefined() ||
       (enable.isConstant() && !enable.getConstant().getValue().isZero()))
     mergeLatticeValue(regReset, srcValue);
+
+  if (enable.isConstant() && enable.getConstant().getValue().isZero())
```

Comment:
nit: can  you rename the enable variable?  It's not the register enable, it's the reset signal.